Scripts/99 SL-ThemePrefs.lua

```diff
@@ -321,6 +320,11 @@ SL_CustomPrefs.Get = function()
 				THEME:GetString("ThemePrefs", "Never"),
 			},
 			Values = { "Always", "Sometimes", "Never" }
+		},
+		PrioritizeLocalLeaderboard = {
+			Default = false,
```

Comment:
Let's just always put the local leaderboard at the end and remove this theme pref. We can later reconsider for mainline if we actually require a preference.